en/docs/setup/setting-up-separate-databases-for-clustering.md

```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
 can do when clustering WSO2 Identity Server.
 
 !!! tip "Before you begin"
-    
-    Creating separate databases as shown below **is not actually required and can be skipped.** Instead, you can point all the datasources given below to a single database. This will NOT make a difference in performance. To do this, see the steps given within the flow of the deployment pattern setup in the [Configuring the
-    datasources](../../setup/configuring-the-datasources). 
-    
-    However, if you do want to separate the data logically into separate
-    databases, you can follow the steps given below.
-    
+    Creating separate databases as shown below **is not mandatory**. Instead, 
+    you can point all the following datasources to the dafault database. This will NOT make a difference 
+    in performance. To do this, see the [Configuring the datasources](../../setup/configuring-the-datasources). 
 
 WSO2 Identity Server uses a database to store information such as
-user management details and registry data. All nodes in the cluster must
-use one central database for config and governance registry mounts. By
+user management details and identity data. By
 default, each WSO2 product is shipped with an embedded H2 database that
 works for all types of data.
 
-!!! warning "Embedded H2 is not recommended in production"
-    
-    The embedded H2 database is NOT recommended in enterprise testing and
+!!! warning "Embedded H2 is NOT RECOMMENDED in production"
+    The embedded H2 database is **NOT RECOMMENDED** for enterprise testing and for
```

Comment:
Please remove for

Reply:
Fixed: 55e7088941e0c75d2805c48dc949bcff9db0324c